app/javascript/src/utilities/utils.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import {fetch as fetchPolyfill} from 'whatwg-fetch'
+import 'core-js/es6/promise'
+import React from 'react'
+
+function CSRFToken ({win = window}: {win?: Window}) {
+  const getMetaContent = meta => win.document.head.querySelector(`meta[name~=${meta}][content]`).content
+
+  try {
+    const props = {
+      name: getMetaContent('csrf-param'),
+      value: getMetaContent('csrf-token'),
+      type: 'hidden'
+    }
+    return (
+      <input {...props} />
```

Comment:
I don't think we gain any readability by defining standalone props here:
```suggestion
      <input
        name={getMetaContent('csrf-param')}
        value={getMetaContent('csrf-token')}
        type='hidden'
      />
```